.github/main.workflow

```diff
@@ -8,11 +8,6 @@ workflow "pull request" {
   resolves = ["docs:status"]
 }
 
-workflow "everyday" {
-  on = "schedule(59 14 * * *)"
-  resolves = ["dist:nuget"]
-}
-
```

Comment:
Don't need to build nightly builds anymore?

Reply:
No, it's a mistake.  I'm going to fix it.

Reply:
Fixed and amended: abd4ac6068546dfc2d58ea5ddf50e0ee61925e7f.